src/System.ValueTuple/tests/System.ValueTuple.Tests.csproj

```diff
@@ -7,7 +7,9 @@
     <ProjectGuid>{CBD5AE8D-8595-48E2-848F-1A3492A28FDB}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AssemblyName>System.ValueTuple.Tests</AssemblyName>
-    <NugetTargetMoniker>.NETStandard,Version=v1.3</NugetTargetMoniker>
+    <NugetTargetMoniker Condition="'$(TargetGroup)' == 'netstandard1.3'">.NETStandard,Version=v1.3</NugetTargetMoniker>
```

Comment:
Don't need to set it for netstandard1.3 as it will get defaulted from the TargetGroup.